dashboard/app/controllers/api/v1/section_instructors_controller.rb

```diff
@@ -0,0 +1,94 @@
+require 'cdo/firehose'
+
+class Api::V1::SectionInstructorsController < Api::V1::JSONApiController
+  # Returns list of current user's SectionInstructor records
+  # GET /section_instructors
+  def index
+    section_instructors = SectionInstructor.where(instructor_id: current_user&.id)
+
+    render json: section_instructors, each_serializer: Api::V1::SectionInstructorSerializer
+  end
+
+  # Returns SectionInstructor records for the given section
+  # GET /section_instructors/:id
+  def show
+    section = Section.find(params.require(:id))
+    authorize! :manage, section
+    section_instructors = SectionInstructor.where(section: section)
+
+    render json: section_instructors, each_serializer: Api::V1::SectionInstructorSerializer
+  end
+
+  # Adds a new teacher as an invited instructor
+  # POST /section_instructors
+  def create
+    section = Section.find(params.require(:section_id))
+    authorize! :manage, section
+
+    # Enforce maximum co-instructor count (the limit is 5 plus the main teacher
+    # for a total of 6)
+    return head :bad_request if section.instructors.length >= 6
```

Comment:
Do we want to make this a constant?

Reply:
I've never really seen the benefit of adding another layer of indirection for a constant that's used only once; would definitely factor it out if it's used again.